### ISSUE TITLE
Link to Android App is Broken

### ISSUE THREAD

**Issue by JBlocklove** [MaxusDev/WelleAPI#13]
The link in the Readme to the Android app is broken. Have y'all just given up on this?

**Comment by hoangminhanhtai** [comment]
Yes it is broken. Hope that Maxus Tech have given up this project yet

**Comment by JBlocklove** [comment]
Their last commit was almost 2 years ago, I think it's safe to say that this project is dead.

**Comment by romandant** [comment]
Hello, I was able to get 2 welle sensors. Unfortunately I am not able to find the (apple/Android) App in the store nor on the links I found in the internet. Could anyone please let me know where I can find the an working apk? Thanks a lot.

**Comment by JBlocklove** [comment]
I think I might still have the app on an old phone, I'll see if I can track it down and pull the apk off for you. Fair warning, I tried getting the Welle to work again around a year or so ago and didn't really get anywhere with it, even with the app, so you may be facing an uphill battle if you've got a project in mind.
